packages/redux/src/categories/types/action.types.ts

```diff
@@ -28,6 +28,29 @@ export type FetchCategoriesAction =
   | FetchCategoriesSuccessAction
   | FetchCategoriesFailureAction;
 
+export interface FetchCategoryRequestAction extends Action {
+  meta: { id: number };
```

Comment:
```suggestion
  meta: { id: Category['id'] };
```

Please, review all locations which define the type of the categoryId value as `number` to be `Category['id']` instead.